binary/script/build_ipxe.sh

```diff
@@ -81,6 +81,14 @@ function customize_aarch_build() {
     sed -i.bak '/^WORKAROUND_CFLAGS/ s|^|#|' "${ipxe_dir}"/src/arch/arm64/Makefile
 }
 
+# Workaround for Broadcom NetXtreme driver bug that causes a hang when
+# trying to download large files. See this iPXE issue for more detail:
+# https://github.com/ipxe/ipxe/issues/1023#issuecomment-1898585257
+function patch_bnxt_rx_buffers() {
+    local ipxe_dir="$1"
+    sed -i.bak 's/\(#define NUM_RX_BUFFERS \).*/\12/' "${ipxe_dir}"/src/drivers/net/bnxt/bnxt.h
```

Comment:
I don't think we need a back up file, do we? `-i.bak`

I'm assuming the compilation is fine even though there is a spacing difference? Regardless, you might think about matching the spacing from the original.

```bash
diff bnxt.h bnxt.h.bak
153c153
< #define NUM_RX_BUFFERS 2
---
> #define NUM_RX_BUFFERS                          8
```

Reply:
Sure, I can remove the backup. I was trying to stick with the form of the other `sed` commands in this file, so I have no preference.

The indentation is purely for readability purposes for the header file. The C preprocessor doesn't care about extra whitespace in object-like macros. If you still want me to match the whitespace I can do that. From my perspective since this is part of a build script readability of the patched code shouldn't really matter.